types.go

```diff
@@ -28,6 +28,18 @@ const (
 	resourceAccountMIRHistory                  = "mirs"
 	resourceAccountAssociatedAddress           = "addresses"
 	resourceAccountAddressWithAssetsAssociated = "addresses/assets"
+	resourceMetadataTxLabels                   = "metadata/txs/labels"
```

Comment:
move resources and new types to the `api_metadata.go`. Keeping defined types and resources in the same `api_{resource}.go` file from now.

Reply:
OK, very interesting approach